CONTRIBUTING.md

```diff
@@ -1,5 +1,48 @@
 # Contributing to Ably Flutter
 
+## Overview
+A [Flutter](https://flutter.dev/) plugin for [Ably](https://www.ably.com), built on top of Ably's [iOS](https://github.com/ably/ably-cocoa) and [Android](https://github.com/ably/ably-java) SDKs. [Web support](https://github.com/ably/ably-flutter/issues/95) will be added in the near future built on top of [ably-js](https://github.com/ably/ably-js).
```

Comment:
I would avoid mentioning web support until we finally decide to do this. I believe there is a chance that instead of using ably-js, we will create a pure Dart implementation.

Reply:
Yeah, I had posted similar message on the channel => https://ably-real-time.slack.com/archives/C03JDBVM5MY/p1709635581276429. 
Then found out `pusher-flutter` also uses `js-interop` => https://github.com/pusher/pusher-channels-flutter along with java and ios plugins same as ably.

Reply:
Though this is something we can decide on in the future.

Reply:
updated !